metagraph/src/cli/config/config.hpp

```diff
@@ -84,7 +84,7 @@ class Config {
     unsigned int bloom_max_num_hash_functions = 10;
     unsigned int num_columns_cached = 10;
     unsigned int max_hull_forks = 4;
-    unsigned int parallel_assemblies = -1;
+    unsigned int parallel_assemblies = -1; // if not set, run one single-threaded assembly per thread
```

Comment:
So why is it not 1 by default?

Reply:
Given N threads, the algorithm parallelizes better if you have N assemblies on one thread each, so I made that the default.

Though I agree that there is the problem that you don't know the number of experiments in advance (unless I change the specification of label-mask-file to start with a single number being the number of experiments), so this default option doesn't work if you just want to run one experiment

Reply:
Maybe the behaviour will be more expected if I just set it to run one assembly at a time (giving it all threads) by default

Reply:
Ah, got it, I misread it first. Maybe write ` // run `parallel` assemblies by default (one per thread).